libs/components/lists/src/lib/modules/repeater/repeater.component.ts

```diff
@@ -200,9 +200,14 @@ export class SkyRepeaterComponent
     // https://github.com/angular/angular/issues/6005
     this.items?.changes.pipe(takeUntil(this.#ngUnsubscribe)).subscribe(() => {
       setTimeout(() => {
-        if (this.items?.last) {
+        if (this.items?.length) {
           this.#updateForExpandMode(this.items.last);
-          this.items.last.reorderable = this.reorderable;
+
+          this.items.forEach((item) => {
```

Comment:
Since this loop is identical to the one later in the constructor, put it in its own private function. Also change the `forEach()` to a `for (const item of items)` loop for easier debugging.